frontend/src/ui/index.tsx

```diff
@@ -157,3 +158,7 @@ export const ellipsisOverflowCss = (lines: number): Record<string, any> => ({
             WebkitLineClamp: lines,
         },
 });
+
+/** Box shadow used on multiple elements in dark mode */
+export const darkModeBoxShadow = { boxShadow: "0 0 10px rgba(0, 0, 0, .3)" };
```

Comment:
Soo on this I spend more time on this than I'd like to admit, trying to find a shadow that works well... and I don't think I necessarily succeeded... Sure, it's (barely) noticeable enough, if your screen is bright enough... but then again, I guess that many dark mode users won't have their screens at max brightness. Meaning I'm not sure if this is really necessary? Though it doesn't hurt either. I am however sure that this shouldn't be an inverted shadow (like a glow... that just looks weird).

Reply:
I would actually prefer to use `grey2` instead of `grey1` on the elements that are using this shadow (nav, series block, dashboard tiles), and then maybe just remove the shadow.

Reply:
As mentioned above, I slightly tweaked some grey tones, including `grey1`. I think the brightness/contrast is sufficient enough with that. I still think we don't need a `box-shadow` but I kept it for now.